curve-amm/src/tests.rs

```diff
@@ -0,0 +1,375 @@
+#[cfg(test)]
+extern crate std;
+#[cfg(test)]
+use std::fmt::Write;
```

Comment:
No need for `cfg(test)`:
```suggestion
extern crate std;
use std::fmt::Write;
```